### DISCUSSION TITLE
September 2023 product updates

### DISCUSSION THREAD

**Discussion by jerirose** [amzn/ads-advanced-tools-docs#139] [Announcements]
We'll be using this thread to highlight releases for September. If you would like to be notified when we post updates, you can subscribe to this discussion. Feel free to ask questions or leave feedback in the thread!

*[August 2023 product updates](https://github.com/amzn/ads-advanced-tools-docs/discussions/122)*
*All releases are also posted on the [advanced tools center](https://advertising.amazon.com/API/docs/en-us/release-notes/index).*

**Comment by jerirose** [comment]
**Original video download is now available on Sponsored Brands media API**

Sponsored Brands API now allows advertisers to download their original video media using the [GET /media/describe](https://advertising.amazon.com/API/docs/en-us/sponsored-brands/3-0/openapi#tag/Media/operation/describeMedia). Advertisers can use `originalMediaUrl` to retrieve their original video media in .MP4 format. Previously, advertisers could only download a preview of the original video using `publishedMediaUrl`.

Going forward, please use the [asset library](https://advertising.amazon.com/API/docs/en-us/creative-asset-library) for all creative management as the [POST /media/upload](https://advertising.amazon.com/API/docs/en-us/sponsored-brands/3-0/openapi#tag/Media/operation/createUploadResource) will be deprecated on 1/31/2024. For further instruction, refer to the API specifications [here](https://advertising.amazon.com/API/docs/en-us/sponsored-brands/3-0/openapi#tag/Media/operation/describeMedia).

**Comment by elizafarley** [comment]
**Amazon DSP campaign signals now available on Amazon Marketing Stream**

Amazon Marketing Stream is expanding to include new Amazon DSP campaign datasets. This includes information on Amazon DSP campaigns, flights, ad groups, and ad group targets.

The Amazon DSP campaign data is split into four datasets: campaigns, flights, ad groups, and ad group targets for each AWS region. To learn more and access this feature, review the Amazon Marketing Stream [Onboarding guide](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-stream/onboarding), and subscribe to the new [Amazon DSP campaign datasets](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-stream/datasets/dsp-campaign-management).

You can create SQS queues for these new datasets using the updated [CloudFormation template](https://github.com/amzn/ads-advanced-tools-docs/blob/main/amazon_marketing_stream/Stream_SQS%20_CF_Template.yaml).

Note: You will need to use the [new Amazon DSP-specific Stream API endpoints](https://advertising.amazon.com/API/docs/en-us/amazon-marketing-stream/openapi#tag/DSP-Stream-Subscription) to subscribe to these datasets.

**Comment by elizafarley** [comment]
**Sponsored Display improves bid recommendations by supporting campaigns with video creatives**

Sponsored Display is launching bid recommendations for video creatives. Advertisers can now customize creatives and receive bid recommendations when using any custom creative, including headline and logo, image, or video. Previously, bid recommendations were only customized for campaigns with static images. With this launch, advertisers receive relevant bid recommendations to ensure their video campaigns have improved inputs to help them reach their audiences.

For full technical details, see our updated [bid recommendations](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Bid-Recommendations) resource. This resource will help find relevant bid suggestions when using either video or image creatives.

**Comment by elizafarley** [comment]
**Sponsored Display optimization rules (beta) API is now available**

We launched a new API, [optimization rules (beta)](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#tag/Optimization-Rules-(beta)), to simplify the campaign management process for advertisers using Sponsored Display. This launch allows advertisers to set performance-based parameters that remove the guess work from selecting an exact bid for each target. This helps advertisers of any size and with any budget achieve their goals with the help of Amazon’s first-party data and machine learning capabilities.

**Comment by elizafarley** [comment]
**Sponsored Products rest-of-search placement bidding adjustments (beta)**

Advertisers using the Amazon Ads API or bulksheets can now apply bid adjustment for rest-of-search placements in their Sponsored Products campaigns. The new placement bid adjustment control for rest of search works same as the adjustments available for top-of-search (first page) and product-page placements. Through the new control, advertisers can adjust their bid for rest of search by up to 900%, same as the other two placements. 

You can view the relevant API contract changes by looking at dynamicBidding.placementBidding.placement in [POST /sp/campaigns](https://advertising.amazon.com/API/docs/en-us/sponsored-products/3-0/openapi/prod#tag/Campaigns/operation/CreateSponsoredProductsCampaigns). Use the new enum PLACEMENT_REST_OF_SEARCH to add an adjustment percentage.

**Comment by elizafarley** [comment]
**Preview: Campaign and ad group export APIs**

Amazon Ads API has launched new export APIs as an update to the existing snapshots functionality. The new APIs currently support asynchronous campaign and ad group exports for Sponsored Products, Sponsored Brands, and Sponsored Display.

These APIs simplify ingestion of Amazon Ads data by providing a common model across different ad products, create parity between asynchronous export APIs and synchronous LIST APIs (including extended data fields), and improve performance over the previous snapshot APIs.

To get started with the export API, see:

- [Migration guide](https://advertising.amazon.com/API/docs/en-us/reference/migration-guides/snapshots-exports) (snapshots to exports)
- [Exports tutorial](https://advertising.amazon.com/API/docs/en-us/guides/exports/get-started)
- [Exports API reference](https://advertising.amazon.com/API/docs/en-us/exports)

We've also updated the [Postman collection](https://github.com/amzn/ads-advanced-tools-docs/blob/main/postman/Amazon_Ads_API.postman_collection.json) to include the new endpoints and example calls. 

**Note**: We’re announcing these APIs in preview so that you can make test calls and start your integration. During preview, these endpoints have a smaller TPS and job queue capacity. Once these APIs are ready for scaled adoption and support their full TPS and job queue capacity, we’ll post a new release note.

If you have any questions or feedback on this new feature, please let us know!

**Comment by elizafarley** [comment]
**Sponsored Display has enhanced the forecasting API by adding improved responses**
 
Sponsored Display has added forecast status to the API to better understand how targeting can affect delivery and scale.

Forecasting helps advertisers understand the impact of their choices during campaign creation with a minimum and maximum range of available impressions inventory for a set of selected advertised products, optimization type, targeting, and bid values. This helps advertisers create campaigns that will deliver results, understanding scale by experimenting with the campaign setup, and forecasting the output of an investment before they get started.

The new forecastStatus is now available through the Sponsored Display including IMPRESSION_TARGETING_TOO_NARROW and IMPRESSION_TARGETING_TOO_BROAD. For full technical details, please see our updated [Sponsored Display documentation](https://advertising.amazon.com/API/docs/en-us/sponsored-display/3-0/openapi#/Forecasts).
